docs/var.tfvars-doc.md

```diff
@@ -66,12 +66,16 @@ To set a pre-defined IPv4 address for the bastion node, make use of the optional
 ```
 bastion                     = {instance_type    = "<bastion-compute-template>", image_id    = "<image-uuid-rhel>",  "count"   = 1,  fixed_ip_v4 = "<IPv4 address>"}
 ```
-
+For bastion HA with pre-defined IPs, here the `fixed_ip_v4` will be the VIP for bastions:
+```
+bastion                     = {instance_type    = "<bastion-compute-template>", image_id    = "<image-uuid-rhel>",  "count"   = 2,  fixed_ip_v4 = "<IPv4 address>", fixed_ips = ["<IPv4 address>", "<IPv4 address>"]}
```

Comment:
can't we name it `fixed_vip` ?

Reply:
As per this change, the fixed_ip_v4 will be used for bastion in case of non HA. So naming it vip does not make sense for backward compatibility.

Reply:
If we want to rename to vip then fixed_ips needs to be used even for non HA.

Reply:
yeah I see your point :-(